ansible/roles/kubernetes-addons/tasks/main.yml

```diff
@@ -13,7 +13,7 @@
 
 - name: Make sure the system services namespace exists
   get_url:
-    url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/saltbase/salt/kube-addons/namespace.yaml
+    url=https://raw.githubusercontent.com/kubernetes/kubernetes/master/{{ kube_addons_download_source_dir }}/namespace.yaml
     dest="{{ kube_config_dir }}/addons/"
```

Comment:
dest is hard coding to /opt/namespace.yaml in kube-addons.sh, it need to adjust accordingly.

https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/kube-addons.sh#L68